src/Components/table.js

```diff
@@ -0,0 +1,58 @@
+import React, {useState} from 'react';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+import {Checkbox} from "@material-ui/core";
+import styled from 'styled-components'
+
+const TableHeadCellStyled = styled(TableCell)`
+        background: #CED2AA;  
+        font-size : large;
+`;
+
+const TableBodyCellStyled = styled(TableCell)`
+        font-size: 14px;
+`;
+
+const TableStyled = styled(Table)`
+    min-width: 100px;
+    margin-top: 10px;
+`;
+
+export default function ListOfUsers() {
+
+    const [users] = useState([])
```

Comment:
I'd suggest to initiate the variable with `null` instead of empty array. Because it assumes that this variables value is going to be used in loop.